CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing
+
+You **always** want to look at this file *before* contributing. In here you should find
+steps that you need to take to set up your development environment as well as instructions
+for coding standards and contributing guidelines.
+
+
+## Set up instructions
+
+You're going to have to do those things to contribute to this project.
+
+1. Fork the repo
+2. Clone your fork
+3. Create upstream by running `git remote add upstream https://github.com/KuriusMTL/HackItForward.git`. You will then be able to run `git pull upstream master`to pull any changes from the the original repository since you last forked it).
+4. Create a branch by running `git checkout -b <branch-name>`
+5. Run these commands
```

Comment:
We should probably mention Python virtual environments.

Reply:
Can you directly add your suggestions? I will accept the changes you suggest